apps/aecore/lib/aecore/oracle/oracle_state_tree.ex

```diff
@@ -0,0 +1,195 @@
+defmodule Aecore.Oracle.OracleStateTree do
```

Comment:
this is split into sections using comments, consider moving the sections in seperate files

Reply:
I cleaned up the module and there is no need to split in different files

Reply:
Could we move Oracle logic out of this file and have it only handle the actual state tree changes

Reply:
In naming we have the `apply_block_height_on_state!` to apply logic not connected to one specific transaction in the general Naming module

Reply:
Could you be more specific about the logic which should be moved out and where?